src/core/utils.jl

```diff
@@ -34,7 +34,16 @@ function annuity(years::Int, rate_escalation::Real, rate_discount::Real)
         i.e. it is a geometric sum of (1+rate_escalation)^n / (1+rate_discount)^n
         for n = 1, ..., years
     """
-    x = (1 + rate_escalation) / (1 + rate_discount)
+    return annuity_two_escalation_rates(years, rate_escalation, 0.0, rate_discount)
+end
+
+function annuity_two_escalation_rates(years::Int, rate_escalation1::Float64, rate_escalation2::Float64, rate_discount::Float64)
+    """
+        this formulation assumes cost growth in first period
+        i.e. it is a geometric sum of (1+rate_escalation)^n / (1+rate_discount)^n
```

Comment:
this doc string is incorrect

Reply:
it should show how the two escalation (growth) rates are accounted for, like:

(1+g1)^y * (1+g2)^y = (1+g1+g2+g1*g2)^y

Reply:
addressed in 7e198f0